File: Dec/Arm.java
import java.util.Scanner;

public class Arm 
{
    public static void main(String[] args)
     {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter number :");
        int n = sc.nextInt();
         int icnt=0,rem=0,sum=0;
        int temp=n;
        while(n>0)
        {
            icnt++;
            n=n/10;

        }
        n=temp;
        while(n>0)
        {
            int mult=1;
            rem = n%10;
            for(int i=1;i<=icnt;i++)
            {
                mult=mult*rem;
            }
            sum=sum+mult;
            n=n/10;
        }
        if(temp==sum)
        {
            System.out.print("number is Arm");
        }
        else
        {
            System.out.print("number is not Arm");

        }
    }
    
}

File: FEB/Arm.c
#include<stdio.h>
void Display(int No)
{
    int temp=0;
    int icnt=0,mult=0,sum=0;
    temp=No;
    while(No!=0)
    {
        icnt++;
        No=No/10;
    }
   No=temp;
   
    
    while (No!=0)
    {
        mult=1;
        int rev=No%10;
        for(int i=1;i<=icnt;i++)
        {
             mult=mult*rev;
        }
        sum=sum+mult;

       No=No/10;

    }
     
    if(sum==temp)
    {
        printf("Number is Armstrong");
    }
    else
    {
    printf("Number is NOT Armstrong");
    }
    
}

int main()
{
    int ino=0,iret=0;
    printf("Enter the number: ");
    scanf("%d",&ino);
   Display(ino);
   

    return 0;
}